plugin/pki/path_venafi_secrets.go

```diff
@@ -0,0 +1,345 @@
+package pki
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathCredentialsList(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: CredentialsRootPath + "?$",
+		Fields:  nil,
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ListOperation: &framework.PathOperation{
+				Callback: b.pathVenafiSecretList,
+				Summary:  "List all venafi secrets",
+			},
+		},
+		HelpSynopsis:    "",
+		HelpDescription: "",
+	}
+}
+
+func pathCredentials(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: CredentialsRootPath + framework.GenericNameRegex("name"),
+		Fields: map[string]*framework.FieldSchema{
+			"name": {
+				Type:        framework.TypeString,
+				Description: "Name of the authentication object",
+				Required:    true,
+			},
+			"zone": {
+				Type: framework.TypeString,
+				Description: `Name of Venafi Platform or Cloud policy. 
+Example for Platform: testpolicy\\vault
+Example for Venafi Cloud: e33f3e40-4e7e-11ea-8da3-b3c196ebeb0b`,
+				Required: true,
+			},
+			"tpp_url": {
+				Type:        framework.TypeString,
+				Description: `URL of Venafi Platform. Example: https://tpp.venafi.example/vedsdk`,
+				Deprecated:  true,
+			},
+			"url": {
```

Comment:
we should update the url description, to say that it works for cloud url too and remove the /vedsdk

Reply:
URL description updated to indicate that it also replaces cloud_url field

Reply:
Please change "URL of Venafi Platform" to "URL of Venafi API endpoint" and remove "It replaces tpp_url and cloud_url."  We don't need to bring awareness of deprecated parameters in the context of the parameter that replaced them.